docs/Changelog.md

```diff
@@ -10282,3 +10282,46 @@ This version of the operator has been available since version 10 of the default
 
 This version of the operator has been deprecated since version 10 of the default ONNX operator set.
 
+## Version 11 of the default ONNX operator set
+### <a name="Unique-11"></a>**Unique-11**</a>
+
+  Finds all the unique values (deduped list) present in the given input tensor. This operator returns 3 outputs. 
+  The first output tensor 'y' contains all of the unique elements of the input, sorted in the same order that they occur in the input.
+  The second output tensor 'idx' is the same size as the input and it contains the index of each value of the input in 'y'.
+  The third output tensor 'counts' contains the count of each element of 'y' in the input.
+  
+  Example:
+    input_x = [2, 1, 1, 3, 4, 3]
+    output_y = [2, 1, 3, 4]
+    output_idx = [0, 1, 1, 2, 3, 2]
+    output_counts = [1, 2, 2, 1]
+
+#### Version
+
+This version of the operator has been available since version 11 of the default ONNX operator set.
+
+#### Inputs
+
+<dl>
+<dt><tt>x</tt> : T</dt>
+<dd>A 1-D input tensor that is to be processed.</dd>
+</dl>
+
+#### Outputs
+
+<dl>
+<dt><tt>y</tt> : T</dt>
+<dd>A 1-D tensor of the same type as 'x' containing all the unique values in 'x' sorted in the same order that they occur in the input 'x'</dd>
+<dt><tt>idx</tt> : tensor(int64)</dt>
```

Comment:
tf.unique output idx supports both int64 and int32, but we only support int64. Shall we need support int32 also?

Reply:
@linkerzhang - I initially preferred to keep it to just INT64 only, but what's your opinion ?

Reply:
int64 is good enough, I think.